web/static/js/components/surveys/SurveyStatus.jsx

```diff
@@ -36,57 +37,52 @@ class SurveyStatus extends PureComponent {
 
     switch (unit) {
       case 'second':
-        return t('{{count}} second from now', {count: number})
+        return t('{{count}} second from now', { count: number })
       case 'minute':
-        return t('{{count}} minute from now', {count: number})
+        return t('{{count}} minute from now', { count: number })
       case 'hour':
-        return t('{{count}} hour from now', {count: number})
+        return t('{{count}} hour from now', { count: number })
       case 'day':
-        return t('{{count}} day from now', {count: number})
+        return t('{{count}} day from now', { count: number })
       case 'week':
-        return t('{{count}} week from now', {count: number})
+        return t('{{count}} week from now', { count: number })
       case 'month':
-        return t('{{count}} month from now', {count: number})
+        return t('{{count}} month from now', { count: number })
       case 'year':
-        return t('{{count}} year from now', {count: number})
+        return t('{{count}} year from now', { count: number })
     }
   }
 
   startedFormatter(number, unit, suffix, date, defaultFormatter) {
     const { t, survey } = this.props
 
     if (unit == 'second') {
-      return t('Started {{count}} second ago', {count: number})
+      return t('Started {{count}} second ago', { count: number })
     } else if (unit == 'minute') {
-      return t('Started {{count}} minute ago', {count: number})
+      return t('Started {{count}} minute ago', { count: number })
     } else if (unit == 'hour') {
-      return t('Started {{count}} hour ago', {count: number})
+      return t('Started {{count}} hour ago', { count: number })
     } else if (unit == 'day') {
-      return t('Started {{count}} day ago', {count: number})
+      return t('Started {{count}} day ago', { count: number })
     } else {
-      return t('Started {{date}}', {date: dateformat(survey.startedAt, 'mmm d, yyyy HH:MM (Z)')})
+      return t('Started {{date}}', { date: dateformat(survey.startedAt, 'mmm d, yyyy HH:MM (Z)') })
     }
   }
 
   nextCallDescription(survey, date) {
-    const hour = this.hourDescription(survey, date)
-    if (this.props.short) {
-      return <Trans>Scheduled at {{hour}}</Trans>
-    } else {
-      return <Trans>Next contact <TimeAgo date={date} formatter={this.bindedFormatter} /> at {{hour}}</Trans>
-    }
-  }
-
-  hourDescription(survey, date) {
-    const { timezones } = this.props
-    let locale = Intl.DateTimeFormat().resolvedOptions().locale || 'en-US'
+    const { timezones, t, language } = this.props
     let options = {
       timeZone: timezones.items[survey.schedule.timezone],
+      year: 'numeric',
+      month: 'short',
+      day: 'numeric',
       hour12: true,
       hour: 'numeric'
     }
-    let time = date.toLocaleTimeString(locale, options)
-    return `${time} (${formatTimezone(survey.schedule.timezone)})`
+    let dateString = date.toLocaleDateString(language, options).replace(/[,.]/g, '')
```

Comment:
Why we need to remove `,` and `.`? Some date formats might use `.` as date component separators. I would suggest to `.replace(/a\.m\./, 'AM')` if the goal is to use AM instead of a.m. . Also PM should be handled.

Reply:
These are the new messages (the dots aren't removed any more):

```
Programada para 22 mar. 2021 9 AM (America - Buenos Aires)
Scheduled for Mar 22 2021 9 AM (America - Buenos Aires)
Planifié pour 22 mars 2021 à 9 AM (America - Buenos Aires)
````

Reply:
And... after midday:
```
Programada para 16 mar. 2021 3 PM (America - Buenos Aires)
Scheduled for Mar 16 2021 3 PM (America - Buenos Aires)
Planifié pour 16 mars 2021 à 3 PM (America - Buenos Aires)
```